### PR TITLE
KFSPTS-12079  upgrade to 5/24/2018 version of financials

### PR DESCRIPTION
**Pull request by db483**

### CODE REVIEW

**Comment by db483** [comment]
Thank you very much Chad! I am adding the changes and I have created a user story for [4].

**Comment by db483** [comment]
Hi Chad,

I have added the changes that you suggested. Please review again. Thank you!

File: src/main/resources/edu/cornell/kfs/module/purap/cu-spring-purap.xml [hunk 1]
```diff
@@ -53,9 +53,6 @@
 	    <property name="digestorRulesFileName">
 	       <value>edu/cornell/kfs/module/purap/util/cxml/b2bPunchOutOrderDigesterRules.xml</value>
 	    </property>
-	    <property name="schemaLocation">
-	       <value>${externalizable.static.content.url}/xsd/purap/b2bPunchOutOrder.xsd</value>
```
Comment (cah292) [inline src/main/resources/edu/cornell/kfs/module/purap/cu-spring-purap.xml]: Should this schema location be set to the new classpath-based location, as opposed to just removing it entirely?

**Comment by db483** [comment]
Great catches Chad, I have made the changes.

**Pull request merged**